lib/ads/server_side_ad_manager.js

```diff
@@ -220,6 +202,18 @@ shaka.ads.ServerSideAdManager = class {
     }
   }
 
+  /**
+   * @param {Object} value
+   */
+  onCueMetadataChange(value) {
+    // Native HLS over Safari/iOS/iPadOS
```

Comment:
Do I understand correctly that cue metadata change only happens on live streams?
Please add a more detailed comment, explaining when cue metadata changes occur.

Reply:
I have added a comment explaining the need for this